src/collective/cover/interfaces.py

```diff
@@ -30,14 +32,8 @@ def getSearchResults(self, filter_portal_types, rooted, document_base_url, searc
 
 
 class ICoverUIDsProvider(Interface):
-
-    def getUIDs(self):
-        """Get UUIDs associated with the object.
-            could be the UUID of the object or a
-            list of related UUIDs.
-
-        @return: iterable of UUIDs
-        """
+    """GenericUIDsProvider interface will be removed in collective.cover v1.7."""
+    warnings.warn(__doc__, DeprecationWarning)
```

Comment:
Before continuing, we did two really simple PR's that could be merged: https://github.com/collective/collective.cover/pull/736 and https://github.com/collective/collective.cover/pull/735.

Following the adapter issue, we've created https://github.com/collective/collective.cover/pull/737. If you agree with it, we need it merged and a collective.cover and a brasil.gov.tiles release, since we're upgrading collective.cover in IDGB and changes will be needed in brasil.gov.tiles after these both releases. We're planning on finishing the upgrade of collective.cover to 1.5b1 in IDGB tomorrow.